strings_lab.js

```diff
@@ -0,0 +1,178 @@
+// # String Exercises
```

Comment:
Good attempt on bonuses and other lab problems. I left few feedbacks below which pertain to hard coding and some suggestions for better logics for code. Always trying to find a way to write a code that can accommodate for differing inputs. Ask me or Alejo if you have any questions.